core/graph/src/main/java/uk/gov/gchq/gaffer/operation/export/graph/ExportToOtherAuthorisedGraph.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.operation.export.graph;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import uk.gov.gchq.gaffer.commonutil.Required;
+import uk.gov.gchq.gaffer.data.element.Element;
+import uk.gov.gchq.gaffer.operation.Operation;
+import uk.gov.gchq.gaffer.operation.export.ExportTo;
+import uk.gov.gchq.gaffer.operation.io.MultiInput;
+import uk.gov.gchq.gaffer.operation.serialisation.TypeReferenceImpl;
+
+public class ExportToOtherAuthorisedGraph implements
+        Operation,
+        MultiInput<Element>,
+        ExportTo<Iterable<? extends Element>> {
+
+    @Required
+    private String graphId;
+    private Iterable<? extends Element> input;
+    private String parentSchemaId;
```

Comment:
Please can we make this a list of parent schema ids like we have in ExportToOtherGraph.